jobsdb/jobsdb.go

```diff
@@ -365,6 +365,11 @@ type JobStatusT struct {
 	WorkspaceId   string          `json:"WorkspaceId"`
 }
 
+type ConnectionDetailsT struct {
```

Comment:
We no longer use `T` suffix for structs 

```suggestion
type ConnectionDetails struct {
```